cmd/root.go

```diff
@@ -27,11 +27,9 @@ func init() {
 
 	validateCmd.Flags().StringVar(&flags.validate.policyPath, "policy-path", "", "Path to directory containing validation policies")
 	validateCmd.MarkFlagRequired("policy-path")
-	validateCmd.Flags().StringVar(&flags.validate.project, "project", "", "Provider project")
-	validateCmd.MarkFlagRequired("project")
+	validateCmd.Flags().StringVar(&flags.validate.project, "project", "", "Provider project override")
```

Comment:
nit: can we expand on the override behavior a bit in the flag documentation?

Reply:
Updated